### PR TITLE
Retry create_tags call

### PR DESCRIPTION
**Pull request by a1exsh**
Fight eventual consistency issues in AWS API.

### CODE REVIEW

**Comment by tor-vs-floki** [comment]
👍

File: bubuku/aws/ec2_node_launcher.py [hunk 1]
```diff
@@ -69,14 +69,24 @@ def _launch_instance(self, ip: str, subnet: dict, ami: object, security_group_id
...
+                        {'Key': 'Name', 'Value': self._cluster_config.get_cluster_name()},
+                        {'Key': 'StackName', 'Value': self._cluster_config.get_cluster_name()}
+                    ])
+                )
+                break
+
+            except Exception as e:
+                _LOG.error('Failed to create instance tags', exc_info=e)
+                time.sleep(5)
+                attempts -= 1
```
Comment (antban) [inline bubuku/aws/ec2_node_launcher.py:89]: So the exception will be effectively swallowed after a number of attempts?

**Review by antban — CHANGES_REQUESTED** [review]

File: bubuku/aws/ec2_node_launcher.py (hunk 1 above)
Comment (a1exsh) [reply bubuku/aws/ec2_node_launcher.py:89]: Yes.  Do you suggest we throw instead if all attempts failed?

**Comment by tor-vs-floki** [comment]
👍

**Pull request merged**